spec/factories/conversation.rb

```diff
@@ -0,0 +1,4 @@
+FactoryBot.define do
+  factory :conversation do
+  end
+end
```

Comment:
If this is empty, remove the file.

Reply:
I need this when calling it from factories that have an association with conversation. Otherwise initializing anything that contains a conversation fails with 

>  Trait not registered: "conversation"